src/App.jsx

```diff
@@ -14,45 +16,90 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_FIELD_ALPHABET = 'byAlphabet';
+const SORT_FIELD_LENGTH = 'byLength';
+
+function preparedGoodsFromServer(goods, sortField, reverse) {
```

Comment:
please use more clear names
```suggestion
function getSortedGoods(goods, sortType, isReversed) {
```